components/InputFormHome.js

```diff
@@ -90,7 +90,6 @@ export default function AddFacts({ addNewData, showMe, setShowMe }) {
 const Fieldset = styled.div`
```

Comment:
It seems to be the same styled div as in the expenses form. You could export the constant and import it here.